packages/js-sdk/src/loadContracts.d.ts

```diff
@@ -2,6 +2,32 @@ import { ethers, Contract as EthersContract, utils } from "ethers";
 import type { Contract as Web3Contract } from "web3-eth-contract";
 import TruffleContract from "@truffle/contract";
 import type Web3 from "web3";
+import type { Contract } from 'ethers'
+
+type SuperfluidContractNames =
+  | 'ERC20WithTokenInfo'
+  | 'IConstantFlowAgreementV1'
+  | 'IERC20'
+  | 'IInstantDistributionAgreementV1'
+  | 'IResolver'
+  | 'ISETH'
+  | 'ISuperAgreement'
+  | 'ISuperToken'
+  | 'ISuperTokenFactory'
+  | 'ISuperfluid'
+  | 'ISuperfluidGovernance'
+  | 'SuperfluidLoader'
+  | 'TestToken'
+  | 'TokenInfo'
+  | 'UUPSProxiable';
+
+export type SuperfluidContractObject = {
+  abi: any;
```

Comment:
not a super useful change, but I believe we can get the type of this via:
`import type { ContractInterface } from "ethers";`
and
`abi: ContractInterface`.

again, this may break something with the tests that I am not aware of, but worth a try.